src/Sulu/Component/Content/Compat/Block/BlockProperty.php

```diff
@@ -45,7 +45,6 @@ public function __construct(
             $mandatory,
             $multilingual,
             $maxOccurs,
-            $maxOccurs,
```

Comment:
Maybe we could use named arguments here. That would make errors easier to spot and prevent this from happening in the future.

Reply:
phpstan would also spot it so the error was just the big phpstan baseline update.

Reply:
Ah okay.

Reply:
its just that no CI runs after we merged back to 2.6. Some make 2.5-2.6 backmerge branches think that would be overhead. so typed properties would in this case not catched the error and the current CI did already catch it ;)